frontend-react/src/views/Drives/components/DrivesList/index.js

```diff
@@ -0,0 +1,168 @@
+import React, { useState } from 'react';
```

Comment:
React import is redundant as from version 17 you no longer need to explicitly import the React module in files containing React components

Reply:
Thank you, removed